src/test/java/com/artipie/http/VersionSliceTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2020-2021 artipie.com
+ * https://github.com/artipie/artipie/LICENSE.txt
+ */
+package com.artipie.http;
+
+import com.artipie.ArtipieProperties;
+import com.artipie.asto.test.TestResource;
+import com.artipie.http.hm.RsHasBody;
+import com.artipie.http.hm.RsHasStatus;
+import com.artipie.http.hm.SliceHasResponse;
+import com.artipie.http.rq.RequestLine;
+import com.artipie.http.rq.RqMethod;
+import com.artipie.http.rs.RsStatus;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Properties;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Tests for {@link VersionSlice}.
+ * @since 0.21
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
+ */
+final class VersionSliceTest {
+    @Test
+    void returnVersionOfApplication() throws IOException {
+        final Properties properties = new Properties();
```

Comment:
The idea of moving `Properties` object to constructor of `VersionSlice` was to separate tests of HTTP object and properties-loading object. In test for `VersionSlice` we can construct `Properties` file without loading it from resource, just creating on the fly:
```java
var prop = new Properties();
prop.put("artipie.version", "1.0-test");
MatcherAssert.assert(new VersionSlice(prop), ...);
```